runners/jet-experimental/build.gradle

```diff
@@ -19,6 +19,7 @@
 import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
+plugins { id 'maven-publish' }
 applyJavaNature()
```

Comment:
You should be able to load the plugin via the Java nature. See the comment further below.

Reply:
I will remove the whole option of generating examples from sources. Generating them from snapshots is more than enough (and more convenient).